### ISSUE TITLE
Don't load entire database for creating temp video segments

### ISSUE THREAD

**Issue by youralien** [santosfamilyfoundation/TrafficCloud#80]
When creating a video segment, we load the entire database and then only use the trajectories for those frames.

If one creates a table `object_to_frames`

```sql
create table object_to_frames as select distinct object_id, frame_number from objects_features inner join positions on objects_features.trajectory_id=positions.trajectory_id;
```
(which takes 21 seconds)

and then query to get the objects that within the frame limits, i.e. objects between frames 5000 - 5400.

```sql
select distinct object_id from object_to_frames where frame_number > 5000 and frame_number < 5400;
```
(which takes 20ms)

**Comment by youralien** [comment]
Going to add these to storage.py...

```python
def createObjectsToFramesTable(cursor):
    cursor.execute("CREATE TABLE IF NOT EXISTS objects_to_frames AS SELECT DISTINCT (object_id, frame_number) FROM objects_features INNER JOIN positions ON objects_features.trajectory_id=positions.trajectory_id")

def getObjectsInFrameLimits(cursor, firstFrameNum, lastFrameNum):
    cursor.execute("SELECT DISTINCT object_id FROM object_to_frames where frame_number > ? and frame_number < ?", (firstFrameNum, lastFrameNum))
    objects = []
    for row in cursor:
        objects.append(row[0])
    return objects
```

**Comment by youralien** [comment]
since an object_trajectories table gets created, no need to create a new objects_to_frame table
```python
def getObjectsInFrameLimits(cursor, firstFrameNum, lastFrameNum):
    cursor.execute("SELECT DISTINCT object_id FROM object_trajectories where frame > ? and frame < ?
", (firstFrameNum, lastFrameNum))
    objects = []
    for row in cursor:
        objects.append(row[0])
    return objects
```

**Comment by dcelik** [comment]
Resolved.

**Comment by runnersaw** [comment]
Fixed in #91
